### ISSUE TITLE
Flash when maximizing or restoring the window

### ISSUE THREAD

**Issue by leonasdev** [illixion/vscode-vibrancy-continued#28]
https://user-images.githubusercontent.com/39915562/193455418-e646070d-65f7-4e8a-9fea-9718bccab0c5.mp4

happened with any themes.

**Comment by illixion** [comment]
Happens on macOS as well, haven't looked into how this issue can be fixed, but I'd appreciate any help with finding the cause of this.

**Comment by 0xCCY** [comment]
> Happens on macOS as well, haven't looked into how this issue can be fixed, but I'd appreciate any help with finding the cause of this.

I used to think that it was caused by Windows only. But since you're macOS and it happens, it's likely due to VSCode itself.

Upon closer look, I found this:
https://github.com/illixion/vscode-vibrancy-continued/blob/eae6fc5da305fd272b2e3f84d8f96715ca5e71ca/runtime/index.js#L87-L90

This was done to override VSCode's updateBackgroundColor function upon any repaint since we are setting the background to transparent after the DOM is ready not within the theming itself, so the background will be update to the theme's color for a brief second before turning transparent.

I have tried setting the interval to 1ms and it seems working without flicker. 

Submitted a PR (#35) that includes a new config setting to change the refresh interval because I'm not particularly sure if setting it to a value too low will cause any performance issues or side effect. So let's have it as an experimental config setting first, if it's stable we could set it to 10ms in the code.
